DEVELOPER_GUIDE.md

```diff
@@ -55,6 +56,10 @@ In the base OpenSearch Dashboards directory, run
 
   Starts OpenSearch Dashboards and includes this plugin. OpenSearch Dashboards will be available on `localhost:5601`.
 
+- `yarn start --no-base-path --no-watch --server.host="0.0.0.0"`
```

Comment:
Can you move this to Test section, and call out `--server.host="0.0.0.0"` is needed for cypress test to connect to the dashboard server.